### DISCUSSION TITLE
Best way to determine radiant heat flux at head height using CFast 7.7.4

### DISCUSSION THREAD

**Discussion by murphy2626** [firemodels/cfast#2075] [General]
Hi, I'm looking into where occupant tenability criteria are exceeded in a model I've created in CFast. Could anyone tell me how I would determine the radiant heat flux at head height (say 2m) using CFast. I've tried using the 'targets' tab but I am not sure if this is correct given I have to specify the thermal properties of the object / target. Thank you in advance.

**Comment by drjfloyd** [comment]
That is the correct method. If you look at the target radiative gauge heat flux output then it doesn't matter what properties you give (need to  use the validation output to get that quantity).

**Reply by murphy2626** [reply]
Thank you!
